mm2src/coins_activation/src/tendermint_with_assets_activation.rs

```diff
@@ -199,6 +199,13 @@ impl PlatformWithTokensActivationOps for TendermintCoin {
         .await
     }
 
+    async fn enable_global_nft(
+        &self,
+        _activation_request: &Self::ActivationRequest,
+    ) -> Result<Option<MmCoinEnum>, MmError<Self::ActivationError>> {
+        todo!()
+    }
```

Comment:
> I am surprised this hasn't been caught by anyone in previous reviews. I guess the failing CI tests are the cause of that, it shall be fixed soon though.
P.S. The 2 tests below fail here and not in dev, until CI is fixed everyone should compare failing tests in dev to their PRs and not make PRs ready for review until they are the same.

Simply because tests are almost all the time fails :/

Reply:
Additionally, isn't `enable_global_nft` is platform specific? I don't think we should have it in the activation traits?

Reply:
> Additionally, isn't `enable_global_nft` is platform specific? I don't think we should have it in the activation traits?

https://github.com/KomodoPlatform/komodo-defi-framework/pull/2049#discussion_r1504222247
As was mentioned we will see, if we need to separate nft ops from coins and(or) add nfts in coins config, we will do the refactoring of course. 

The plan was to start nft support from EVM and but try to integrate functionality into coins logic. We wanted to re-utilize the existing code. But may be we were wrong. When we need to add new nft protocol (not evm) or move from moralis, API, we can re-think our strategy having new information.

Reply:
@shamardy Thanks for the catch!
did this implementation, which shows that these platform coins currently dont have nfts.
```rust
    async fn enable_global_nft(
        &self,
        _activation_request: &Self::ActivationRequest,
    ) -> Result<Option<MmCoinEnum>, MmError<Self::ActivationError>> {
        Ok(None)
    }
```